src/components/AddNewBookForm/_AddNewBookForm.scss

```diff
@@ -1,89 +1,112 @@
-.submit-book {
-  margin-top: $padding-xs;
-  width: 166px;
-  height: 47px;
-  border-radius: 19px;
-  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.5);
-  background-color: $black;
-  font-size: 15px;
-  color: $white;
-  text-align: center;
-}
-
-.input-description {
-  height: 100px;
-  width: 100%;
-}
+$description-height: 200px;
 
-.add-book {
-  width: 100%;
+.upload-progress.redirect {
+  transition: all 0.3s;
   display: flex;
-  margin-left: auto;
-  margin-right: auto;
-  justify-content: space-between;
-  flex-wrap: wrap;
-  align-items: flex-start;
-}
-
-.add-book-container {
-  flex: 100%;
-  margin-top: $padding-sm;
-  float: left;
-  margin-left: auto;
-  margin-right: auto;
-
-  @include media-breakpoint(sm) {
-    flex: 50%;
-    float: left;
-    margin-left: auto;
-    margin-right: auto;
+  justify-content: center;
+  align-content: center;
+  position: fixed;
+  left: 30%;
+  top: 40%;
+  background: black;
+  width: 40%;
+  height: 15%;
+  opacity: 0.6;
+  border-radius: 30px;
+  > h1 {
+    color: white;
+    font-size: 25px;
+    align-self: center;
   }
 }
-
-.add-book input {
-  width: 100%;
-}
-
-.add-book select {
-  border: 1px solid #ccc;
-  width: 100%;
-  border-radius: 3px;
-  overflow: hidden;
-  background: #fafafa no-repeat 90% 50%;
-}
-
-.add-book select {
-  padding: 5px 8px;
+.upload-progress.on {
+  > h1 {
+    display: none;
+  }
   width: 100%;
-  // border: none;
-  box-shadow: none;
-  background: transparent;
-  background-image: none;
-  -webkit-appearance: none;
+  transition: all 1.8s ease;
+  position: fixed;
+  top: $navbar-height;
+  height: 4px;
+  box-shadow: 0 3px 12px rgba(4, 5, 5, 0.5);
```

Comment:
We should consider creating some box-shadow variables, since they come back in multiple places and it is nice to be consistent.

Reply:
I don't mind if we do not do this for now though.

Reply:
Didn't realize variable is also applicable for shadow. 
Done.